DirectEmployers/MyJobs
# [PD-3299] Change workflow state to workflow status

## What's this PR do?
Change workflow state to read workflow status

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the NUO app

## How should this be manually tested?
Pull down repository. Go into NUO and on the filter side you can see the filter name is changed from workflow state to workflow status

## Any background context you want to provide?
